spinnaker.py

```diff
@@ -899,6 +901,27 @@ def main():
 
             flatMetadataObjs.append(metaObj)
 
+    redwood_host = os.environ['REDWOOD_ENDPOINT']
+    ctx = ssl.create_default_context()
+    ctx.check_hostname = False
+    ctx.verify_mode = ssl.CERT_NONE
+    has_existing_bundle_id = False
+
+    # Checks if bundle_id/gnos_id exists in the
+    for fmo in flatMetadataObjs:
```

Comment:
Try to add more comments for each significant block of code. e.g. what the for loop does, what each chunk within the for loop block does, etc.